### PR TITLE
En 1610 datatable endpoints

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/web/config/ConfigController.java [hunk 1]
```diff
@@ -221,38 +235,80 @@ public void setConfigValidator(ConfigValidator configValidator) {
...
         }
 
         return new ResponseEntity<>(new RestResponse(overrides), HttpStatus.OK);
     }
 
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(value = "/kiebpm/datatableWidget/{configId}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<RestResponse> getDataTableWIdgetConfig(@PathVariable int configId) {
+
+        DatatableWidgetConfigDto dto = this.getKieBpmService().getDataTableWIdgetConfig(configId);
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/web/config/ConfigController.java]: Add controller level logging to all of these methods at debug. Both input and output

File: plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/jpkiebpmServicesConfig.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/jpkiebpmServicesConfig.xml]: What is this file needed for? Are the services not picked up as part of the E5 package scans?

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/web/config/ConfigController.java (hunk 1 above)
Comment (paddeo) [reply plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/web/config/ConfigController.java]: added.

File: plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/jpkiebpmServicesConfig.xml (hunk 2 above)
Comment (paddeo) [reply plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/jpkiebpmServicesConfig.xml]: some managers have already been injected but others are new.

**Comment by paddeo** [comment]
closed because #271 contains its commits
